### PR TITLE
OKTA-567519 Configurable Rate Limits for OAuth 2.0 Apps

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?**  Updates to the Rate Limit dashboard page and the Configure OAuth 2.0 Guides with information on the new feature Application Rate Limits.
- **Is this PR related to a Monolith release?** 2023.**3**.0 (delayed from Feb release)

### Resolves:

* [OKTA-567519](https://oktainc.atlassian.net/browse/OKTA-567519)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ Create an OAuth 2.0 service app integration using the Admin Console.
 
 4. Enter a name for your app integration and click **Save**.
 
+5. Optionally, click the **Application rate limits** tab to adjust the percentage rate limit capacity for this service application. By default, each new application has this percentage set at 50%.
```
Comment (isaacdurham-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md:58]: Should this be `Optional. `?

`percentage rate limit capacity` seems a little unclear.

`has this percentage set at` also seems a little informal. Maybe something like "sets the rate limit capacity to 50%"?

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md [hunk 2]
```diff
@@ -194,11 +194,11 @@ The timeline graph shows use over the last week. When a rate violation occurs, y
...
 ##### Bar graph
 
-The bar graph shows either the top 10 IP addresses or top 10 API tokens that are consuming the most traffic for the API. This view enables you to check if there are IP addresses or API tokens that are creating the spike in traffic.
+The bar graph shows either the top 10 IP addresses, top 10 API tokens, or top 10 OAuth 2.0 applications that are consuming the most traffic for the API. This view enables you to check if there are IP addresses, API tokens, or applications that are creating the spike in traffic.
 
-Display the desired bar graph by selecting **Top Requests by IP** or **Top Requests by API Token** in the dropdown menu.
+Display the desired bar graph by selecting **Top Requests by IP**, **Top Requests by API Token**, or **Top Requests by Application** in the dropdown menu.
 
-Click an IP address or API token bar once to view the traffic for that IP address or API token on the line graph.
+Click an IP address, API token, or Application bar once to view the traffic for that IP address, API token, or Application on the line graph.
```
Comment (isaacdurham-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:201]:
```suggestion
Click an IP address, API token, or Application bar to view the associated traffic on the line graph.
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md](https://github.com/okta/okta-developer-docs/blob/6dfeba1a21d9450d34aea564359c281cb09407a5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-oauth-for-okta-serviceapp%2Fmain%2Findex.md) | 77 | 122 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/98415283-4667-4a9c-bd26-14ba34181462) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md](https://github.com/okta/okta-developer-docs/blob/6dfeba1a21d9450d34aea564359c281cb09407a5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-oauth-for-okta%2Fmain%2Findex.md) | 71 | 125 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8eb9ccbb-8c9c-4dbb-8e2d-bbe8c05c90cf) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md](https://github.com/okta/okta-developer-docs/blob/6dfeba1a21d9450d34aea564359c281cb09407a5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-dashboard%2Findex.md) | 76 | 135 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c9ebaff2-2702-4806-8a48-dca1c5e1aa25) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.ddb333e9-8759-42fd-a532-8cc4a7d33b96?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by isaacdurham-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md](https://github.com/okta/okta-developer-docs/blob/ddc625c8ac7efca8cc23a746c193bb108eff73e9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-oauth-for-okta-serviceapp%2Fmain%2Findex.md) | 77 | 121 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/faa2a8dc-51ac-4a8f-87f8-3309e52131fb) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md](https://github.com/okta/okta-developer-docs/blob/ddc625c8ac7efca8cc23a746c193bb108eff73e9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-oauth-for-okta%2Fmain%2Findex.md) | 71 | 124 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f98614ce-86d0-4d19-8128-08a37c2986b6) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md](https://github.com/okta/okta-developer-docs/blob/ddc625c8ac7efca8cc23a746c193bb108eff73e9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-dashboard%2Findex.md) | 76 | 135 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5ca2fa6e-dda4-41fb-bd19-8d0e1f305037) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.78f9f2c0-38b6-4abe-bdca-713ef505f294?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Pull request merged**